src/dc/nipals.js

```diff
@@ -0,0 +1,64 @@
+
+/**
+ * NIPALS
+ * @param {Matrix} X
+ * @param {*} Y
```

Comment:
Y is a number ? Should be {number}. Small explanation of the option would be nice

Reply:
Y is a column vector to project onto X. If Y is passed then NIPALS works as a PLS instead of a PCA. The help has been updated accordingly.